### PR TITLE
Replace logger with non-blocking and configurable version

### PR DESCRIPTION
**Pull request by a-kosak-mbx**
PR related to the #2215 task.

### CODE REVIEW

**Comment by dgearhart** [comment]
@a-kosak-mbx in the previous PR the log output did not match the original output - has the code been updated in this PR to match the original log output?

File: src/midgard/logging.cc [hunk 1]
```diff
@@ -15,6 +13,60 @@
 
 namespace {
 
+// Highlight special tags according to their leve,
```
Comment (mandeepsandhu) [inline src/midgard/logging.cc]: s/leve/level

**Comment by a-kosak-mbx** [comment]
> @a-kosak-mbx in the previous PR the log output did not match the original output - has the code been updated in this PR to match the original log output?

yes

File: src/midgard/logging.cc (hunk 1 above)
Comment (a-kosak-mbx) [reply src/midgard/logging.cc]: fixed

**Comment by a-kosak-mbx** [comment]
Missing colors: fixed.
Empty files: cannot reproduce, need clarification.

**Comment by kevinkreiser** [comment]
@mookerji recently added glog to the project. @a-kosak-mbx what do you think about replacing our custom logger with glog?

**Comment by mookerji** [comment]
To clarify, didn't actually add this in yet: I've been experimenting with it as part of some other work on the codebase. I separated out glog as a new preview PR linked above.

**Comment by kevinkreiser** [comment]
@mookerji sorry for the confusion, i for some reason though i remembered the gbench thing also adding glog... too many google libs in here as of late..

**Comment by a-kosak-mbx** [comment]
> @mookerji recently added glog to the project. @a-kosak-mbx what do you think about replacing our custom logger with glog?

@kevinkreiser  can we replace our current logger completely? i know that lot's of projects use it
and one more, glog is a bad solution, i've described all cons in #2341

**Comment by kevinkreiser** [comment]
looks like we will be sticking with our basic logging for the time being. closing as stale

**Pull request closed without merging**